pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/errorprone/CloseResourceRule.java

```diff
@@ -189,7 +189,7 @@ private TypeNode getTypeOfVariable(ASTVariableDeclarator var) {
 
     private TypeNode getDeclaredTypeOfVariable(ASTVariableDeclarator var) {
```

Comment:
This is a random place to ask: is it a good idea to use `var` as a variable name?

Reply:
There is no ambiguity with the `var` keyword for type inference if that's what you mean, and in this context it's semantically descriptive (not like `int var1, var2` would be), so to me it's fine

Reply:
Yep, this is what I was asking, I see your point of view, thanks 👌